docs/spec_draft.md

```diff
@@ -642,6 +644,43 @@ operation.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.rsqrt
+
+`stablehlo.rsqrt(operand) -> result`
+
+### Semantics
+
+Performs element-wise reciprocal square root operation on `operand` tensor and
+produces a `result` tensor, implementing the `rSqrt` operation from the IEEE-754
+specification. Numeric precision is implementation-defined.
```

Comment:
Numeric precision is more of a concern for the transcendental function. For the non-transcendental function we do not have to write it.

Reply:
Table 9.1 of IEEE-754 (2008) has a list of "recommended correctly rounded functions." These do not have any guarantees for numeric precision, including `rSqrt`. However, I will remove it for sqrt since `squareRoot` is defined as an arithmetic operation in section 5.4.1.